node/peer.js

```diff
@@ -460,7 +409,6 @@ PreferOutgoingHandler.prototype.getQOS = function getQOS() {
     }
 };
 
-// Consulted depending on the peer state
```

Comment:
Sounds good. Please update the following line:
var outconn = self.peer.getOutConnection();
to 
var outconn = self.peer.getOutConnection(true);